lntest/harness_assertion.go

```diff
@@ -1875,12 +1875,72 @@ func (h *HarnessTest) AssertChannelInGraph(hn *node.HarnessNode,
 
 		return nil
 	}, DefaultTimeout)
+
 	require.NoError(h, err, "%s: timeout finding channel in graph",
 		hn.Name())
 
 	return edge
 }
 
+// AssertChannelInGraphCache asserts a given channel is found in the graph
+// cache.
+func (h *HarnessTest) AssertChannelInGraphCache(hn *node.HarnessNode,
```

Comment:
what if the config setting: `db.no-graph-cache=true` is this then still needed or will it be a NOOP ?

Reply:
Not sure why we call describe graph rather then fetching form the graph cache directly ?

Reply:
> Not sure why we call describe graph rather then fetching form the graph cache directly ?

`DescribeGraph` is correct as the cached graph is what's been used in pathfinding, so it's the de facto graph, albeit it's a cached version of the "true" graph with milliseconds of laggings behind.

when `db.no-graph-cache` is false, then this RPC will query the db directly.